gulpfile.js

```diff
@@ -23,6 +23,7 @@ if (!isCiBuild()) {
 }
 
 const $$ = require('gulp-load-plugins')();
+const dotenv = require('dotenv').config(); // eslint-disable-line
```

Comment:
Because dotenv runs immediately to read `.env` files if available, it does not need to be used after creation. As this violates the `eslint/no-unused-vars` rule, I've disabled it for this line.

Reply:
@ChrisAntaki I tested this, and it still worked fine, so I've removed it in https://github.com/subscriptions-project/swg-js/pull/3422/commits/78a6d31648539ac5a8deccb30e141a8bd6e31323